Classes/Creator/Tca/Table/TcaTableCreator.php

```diff
@@ -108,17 +119,65 @@ private function addNewTcaTypeItems(array $existingTcaTypes, TableInformation $t
 
         /** @var String_ $showItems */
         $showItems = $existingTcaTypes[0]->value->items[0]->value;
-        $existingColumnNames = GeneralUtility::trimExplode(',', $showItems->value, true);
+        $showItemsString = $showItems->value;
 
+        // Normalize: remove all whitespace and check if fields already exist anywhere
+        $normalized = str_replace(["\n", "\r", "\t", ' '], '', $showItemsString);
+        $existingFieldNames = explode(',', $normalized);
+
+        // Collect fields to add
+        $newFields = [];
         foreach ($tableInformation->getColumns() as $columnName => $columnConfiguration) {
-            if (in_array($columnName, $existingColumnNames, true)) {
-                continue;
+            if (!in_array($columnName, $existingFieldNames, true)) {
+                $newFields[] = $columnName;
             }
+        }
+
+        if (empty($newFields)) {
```

Comment:
Please prevent "empty" wherever possible. New fields is an array so just check for
```suggestion
        if ($newFields === []) {
```

Reply:
Done